sale_shipping_info_helper/models/sale_order.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017-To Day Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author: Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    shipping_amount_total = fields.Float(
+        compute="_compute_shipping", digits="Account", store=True
+    )
+    shipping_amount_untaxed = fields.Float(
+        compute="_compute_shipping", digits="Account", store=True
+    )
+    shipping_amount_tax = fields.Float(
+        compute="_compute_shipping", digits="Account", store=True
+    )
+    item_amount_total = fields.Float(
+        compute="_compute_shipping", digits="Account", store=True
+    )
+    item_amount_untaxed = fields.Float(
+        compute="_compute_shipping", digits="Account", store=True
+    )
+    item_amount_tax = fields.Float(
+        compute="_compute_shipping", digits="Account", store=True
+    )
```

Comment:
As a general improvement to the module, perhaps it could be great to add some help descriptions to the fields to better understand which is the purpose for each one. What do you think? 😄